### DISCUSSION TITLE
Videos are scrambled, I want a total refund for my purchase!

### DISCUSSION THREAD

**Discussion by Shyamjidubey** [aclap-dev/video-downloadhelper#1032] [Help and Q&A]
### Your question

I purchased it with the hope that it will be able to download content from website, but all the downloaded videos are scrambled and are not of any use!

The videos are behind a login page of course but it should be able to download them in playable form if they are asking for such a huge amount of $28.5

Its Physics wallah content! also paid content
its www.pw.live, and if u can make your extension support this site i would love that that will be a huge help! 
Thank You.

https://github.com/user-attachments/assets/0582ec08-db15-43ac-b421-8f918e316f86

### Addon & CoApp version and details

version: 9.0.2.12
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"b8780cf2a3fa4ed0"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "concurrent_downloads_max": 10,
    "theme": "light",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_legacy_ui": false,
    "use_sidebar": true,
    "open_count_store": 2,
    "successfull_dl": 11
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

This is not a problem with VDH.  The content you are trying to download is protected by DRM.  Search this forum using `drm` as your search key.  The expressed purpose of DRM is to prevent downloading of copyrighted material.  VDH has never, does not now, & will never break DRM protection.  It would be unlawful to do so.  You should have checked this out before you purchased your VDH license.  The choice of whether to protect something with DRM is entirely at the discretion of the author of the content & the web site on which it is hosted.  This is entirely out of anybody else's hands, out of yours, out of mine, out of VDH's.

On the other hand, VDH works perfectly on lots of content that is not protected by DRM, even content on web sites where you have to pay to get access.  There is lots of unprotected content around.  Surely content protected by DRM is not the only content you want to download.  You might reconsider your refund request.
